src/hearings/models/hearings.enum.ts

```diff
@@ -221,6 +222,15 @@ export enum CancelHearingMessages {
   SELECT_AT_LEAST_ONE_REASON = 'Select at least one valid reason',
 }
 
+export enum HearingChangeReasonMessages {
+  NOT_SELECTED_A_REASON = 'You have not selected a reason for changing the hearing',
+  SELECT_AT_LEAST_ONE_REASON = 'Select at least one valid reason',
+}
```

Comment:
not sure, the way i was guessing that in the future if we want to different messages for cancellation vs change hearing this may be useful, what do you think